project/source/core/Tokenizer.h

```diff
@@ -130,10 +129,11 @@ namespace tokenizer
 				bool first = true;
 				for(int item : items)
 				{
-					if(first)
+					// omit the ',' sign for the first item
+					if(!first)
+						s += Format("%d", item);
+					else
 						first = false;
```

Comment:
teraz nie doda pierwszego elementu do stringa

Reply:
chodzi o to że przekazujesz listę intów 1,2,3 to ci sformatuje to jako {1,2,3}

teraz w ogóle usunąłeś dodawanie ", " więc nie będzie przecinków

Reply:
tak długo edytowałem tego fora aż zapomniałem jak wyglądał na początku